### PR TITLE
Add troubleshooting docs for recordings

### PR DESCRIPTION
**Pull request by NickM-27**

### CODE REVIEW

**Comment by skrashevich** [comment]
maybe, make sense to recommend flags for nfs, for example `noaime,soft,nomtime`?
and system tcp stack tuning: increase tcp rmem and wmem

**Comment by NickM-27** [comment]
> maybe, make sense to recommend flags for nfs, for example `noaime,soft,nomtime`? and system tcp stack tuning: increase tcp rmem and wmem

Maybe, but I think it is difficult absolutely recommend flags because if a flag causes an issue for a particular user. That is why for the fstab part I worded it like:

> Some users found that mounting a drive via `fstab` with the `sync` option caused dramatically reduce performance

because that is not a universal recommendation, just pointing out some things that have caused issues

> and also I think needs to mention that the required shm size dramatically increases when you increase the height and width parameters of `model:`. When using, for example, a deepstack detector, this effect is not obvious

This is already covered in the installation docs and in any case is not related to troubleshooting recordings

**Comment by skrashevich** [comment]
> with thesync option

where I tell about `sync` option?

File: docs/docs/troubleshooting/recordings.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+If the storage is too slow to keep up with the recordings then the maintainer will fall behind and purge the oldest recordings to ensure the cache does not fill up causing a crash. In this case it is important to diagnose why the copy times are slow.
+
+#### Check Storage Type
+
+Mounting a network share is a popular option for storing Recordings, but this can lead to reduced copy times and cause problems. Some users have found that using `NFS` instead of `SMB` considerably decreased the copy times and fixed the issue. It is also important to ensure that the network connection between the device running Frigate and the network share is stable and fast.
+
+#### Check mount options
+
+Some users found that mounting a drive via `fstab` with the `sync` option caused dramatically reduce performance and led to this issue. Using `async` instead greatly reduced copy times.
```
Comment (CoMPaTech) [inline docs/docs/troubleshooting/recordings.md:34]: Suggest rewording;

When using [/etc/fstab](https://en.wikipedia.org/wiki/Fstab) to mount your drives or network shares, make sure you are **not** using the `sync` option as this dramatically reduces performance. As the default for `fstab` is `async` you are not required to add it. For further clarification; In general, though exponentially on network shares, `sync` will wait for acknowledgment of each write, causing additional delays.

**Review by blakeblackshear — APPROVED** [review]
